pysat/utils/_core.py

```diff
@@ -3,8 +3,20 @@
 
 import numpy as np
 import pysat
+import os
 
-
+def new_terrible_function(path=None):
+    """
+    Does terrrrible things
+    """
+    
+    if path is None:
+        path = 37
+        
+     if path is None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [unindent does not match any outer indentation level](https://app.codacy.com/manual/jklenzing/pysat/pullRequest?prid=4912398)